test/tests/ver-1fc/ver-1fc.i

```diff
@@ -0,0 +1,171 @@
+T_SA = 600 # K
+T_SB = 0 # K
+L_A = 40e-2 # m
+L_B = 40e-2 # m
+k_A = 401 # W/m/K
+k_B = 80.2 # W/m/K
+num_nodes = 800 # (-)
+position_measurement = 9e-2 # m
+
+density_Cu = 8960 # kg/m^3
+specific_heat_Cu= 383.8 # J/kg/K
+density_Fe = 7870 # kg/m^3
+specific_heat_Fe = 447.0 # J/kg/K
+
+[Mesh]
+  [whole_domain]
+    type = GeneratedMeshGenerator
+    xmin = 0
+    xmax = ${fparse L_A + L_B}
+    dim = 1
+    nx = ${num_nodes}
+  []
+  [injection_block_1]
+    type = ParsedSubdomainMeshGenerator
+    input = whole_domain
+    combinatorial_geometry = 'x < ${L_A}'
+    block_id = 0
+  []
+  [injection_block_2]
+    type = ParsedSubdomainMeshGenerator
+    input = injection_block_1
+    combinatorial_geometry = 'x > ${L_A}'
+    block_id = 1
+  []
+[]
+
+[Variables]
+  [temperature]
+    initial_condition = 0.0
+    scaling = 1e-6
+  []
+[]
+
+[Functions]
+  [thermal_conductivity_func_Cu]
+    type = ParsedFunction
+    expression = ${k_A}
+  []
+  [thermal_conductivity_func_Fe]
+    type = ParsedFunction
+    expression = ${k_B}
+  []
+[]
+
+[Kernels]
+  [heat]
+    type = HeatConduction
+    variable = temperature
+  []
+  [HeatTdot]
+    type = HeatConductionTimeDerivative
+    variable = temperature
+  []
+[]
+
+[BCs]
+  [lefttemperature]
+    type = DirichletBC
+    boundary = left
+    variable = temperature
+    value = ${T_SA}
+  []
+  [righttemperature]
+    type = DirichletBC
+    boundary = right
+    variable = temperature
+    value = ${T_SB}
+  []
+[]
+
+[Materials]
+  [specific_heat_Cu]
+    type = GenericConstantMaterial
+    block = '0'
+    prop_names = 'density specific_heat'
+    prop_values = '${density_Cu} ${specific_heat_Cu}'
+  []
+  [specific_heat_Fe]
+    type = GenericConstantMaterial
+    block = '1'
+    prop_names = 'density specific_heat'
+    prop_values = '${density_Fe} ${specific_heat_Fe}'
+  []
+  [thermal_conductivity_Cu]
+    type = GenericFunctionMaterial
+    block = '0'
+    prop_values = thermal_conductivity_func_Cu
+    prop_names = thermal_conductivity
+    outputs = exodus
+  []
+  [thermal_conductivity_Fe]
+    type = GenericFunctionMaterial
+    block = '1'
+    prop_values = thermal_conductivity_func_Fe
+    prop_names = thermal_conductivity
+    outputs = exodus
+  []
+[]
+
+[Preconditioning]
+  [SMP]
+    type = SMP
+    full = true
+  []
+[]
+
+[Executioner]
+  type = Transient
+  scheme = bdf2
+  solve_type = NEWTON
+  petsc_options_iname = '-pc_type'
+  petsc_options_value = 'lu'
+  nl_rel_tol = 1e-50
+  nl_abs_tol = 1e-12
+  l_tol = 1e-8
+  dtmax = 5e2
+  end_time = 10000
```

Comment:
How have you confirmed that `t = 10000` is a good steady state time?

Reply:
Good question. Several ways:
(1) I also made it run for longer and got the same results (except numerical fluctuations, basically)
(2) it matches the steady state analytical solution (see figure in documentation)

It could actually be smaller, but 10000 is a nice round number and it's what TMAP7 listed in their documentation (so we might as well use the same since it works).

Reply:
Fine reasons. I mostly wanted to know if we needed to run it for quite that long, but if its consistent with TMAP7, I am OK with that being the metric (considering the whole point is to compare against).